lms/djangoapps/experiments/utils.py

```diff
@@ -261,8 +261,7 @@ def get_experiment_user_metadata_context(course, user):
     has_non_audit_enrollments = False
     try:
         user_enrollments = CourseEnrollment.objects.select_related('course').filter(user_id=user.id)
-        audit_enrollments = user_enrollments.filter(mode='audit')
```

Comment:
I think _audit_enrollments_ is used below in _get_base_experiment_metadata_context_

Reply:
Fixed.